### PR TITLE
Shibam

### PR DESCRIPTION
**Pull request by soulhunter10737**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

File: _data/soulhunter10737/soulhunter10737.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+name: SHIBAM DHAR
+institution: KCCITM
+profile_pic: soulhunter10737.jpg
+quote: Brightness can only be seen in the presemce of darkness.
```
Comment (Mohammed-Shoaib) [inline _data/soulhunter10737/soulhunter10737.md:5]: Typo? Change `presemce` to `presence`.

Comment (soulhunter10737) [reply _data/soulhunter10737/soulhunter10737.md]: Changed it @Mohammed-Shoaib and committed too.

**Comment by soulhunter10737** [comment]
@juanpflores I did not find any form that time. Can you please help me with this.

**Comment by Mohammed-Shoaib** [comment]
@soulhunter10737 Apologies but your first PR was submitted _after_ the deadline (May 26th, 2020 at 12:00 AM PDT) for the swags. :package: Due to this, you couldn't find the form. :disappointed:

**Comment by soulhunter10737** [comment]
@Mohammed-Shoaib  :disappointed: is there any way ?

**Comment by Mohammed-Shoaib** [comment]
@soulhunter10737 I don't think so, I am sorry. :cry: They were already out of swags but extended the deadline based on requests from May 23rd to May 26th as they promised May 26th. So I don't think anything can be done now. :disappointed:

**Comment by juanpflores** [comment]
Unfortunately, the submission for shipping was closed before your first submission :/

**Pull request merged**